internal/template-validator/validator/app.go

```diff
@@ -0,0 +1,101 @@
+package validator
+
+import (
+	"fmt"
+	"net/http"
+
+	flag "github.com/spf13/pflag"
+	"k8s.io/client-go/tools/cache"
+	"kubevirt.io/client-go/log"
+	kubevirtVersion "kubevirt.io/client-go/version"
+
+	"kubevirt.io/ssp-operator/internal/template-validator/service"
+	"kubevirt.io/ssp-operator/internal/template-validator/tlsinfo"
+	"kubevirt.io/ssp-operator/internal/template-validator/version"
+	"kubevirt.io/ssp-operator/internal/template-validator/virtinformers"
+	validating "kubevirt.io/ssp-operator/internal/template-validator/webhooks"
+)
+
+const (
+	defaultPort = 8443
+	defaultHost = "0.0.0.0"
+)
+
```

Comment:
This file is missing the `init()` method that was added to it in the latest PR: https://github.com/kubevirt/kubevirt-template-validator/pull/59

This is the method: 
https://github.com/kubevirt/kubevirt-template-validator/blob/master/pkg/template-validator/app.go#L45

It adds the templates to the scheme. Previously, templates were added by the `init()` method in the imported module `github.com/fromanirh/okdutil/okd`. But it was inefficient to import a whole module for just one line of code.